packages/base/src/processing.ts

```diff
@@ -143,25 +146,13 @@ export async function processSelectedLayer(
     return;
   }
 
-  let processParam: any;
-  switch (processingType) {
-    case 'Buffer':
-      processParam = formValues.bufferDistance;
-      break;
-    case 'Dissolve':
-      processParam = formValues.dissolveField;
-      break;
-    case 'Centroids':
-      processParam = null;
-      break;
-    case 'BoundingBoxes':
-      processParam = null;
-      break;
-    default:
-      console.error(`Unsupported processing type: ${processingType}`);
-      return;
+  if (!ProcessingList.includes(processingType)) {
+    console.error(`Unsupported processing type: ${processingType}`);
+    return;
```

Comment:
This same check exists within `processingFormToParam()`. How do you feel about `throw`ing an error within that function, and catching it here?